<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Manager</class>
 <widget class="QDialog" name="Manager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>599</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="companyLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ðŸ’ª Bulk Club Inc.ðŸ’ª</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>0</y>
     <width>771</width>
     <height>601</height>
    </rect>
   </property>
   <widget class="QWidget" name="salesPage">
    <widget class="QPushButton" name="salesPage_previewButton">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Preview Report</string>
     </property>
    </widget>
    <widget class="QTableView" name="salesPage_tableView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>761</width>
       <height>541</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="customerPage">
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <widget class="QComboBox" name="customerPage_sortBox">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>10</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="customerPage_searchBar">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Enter Customer Name/ID here...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="customerPage_refreshButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ðŸ—˜</string>
     </property>
    </widget>
    <widget class="QPushButton" name="customerPage_searchButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QTableView" name="customerPage_tableView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>761</width>
       <height>541</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="membershipPage">
    <widget class="QToolButton" name="membershipPage_refreshButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ðŸ—˜</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="membershipPage_searchBar">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Enter in a month e.g. November...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="membershipPage_searchButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QTableView" name="membershipPage_tableView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>761</width>
       <height>541</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="inventoryPage">
    <widget class="QToolButton" name="inventoryPage_refreshButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ðŸ—˜</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inventoryPage_searchBar">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Enter Item here...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="inventoryPage_searchButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QComboBox" name="inventoryPage_sortBox">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>10</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QTableView" name="inventoryPage_tableView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>761</width>
       <height>541</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="rebatePage">
    <widget class="QComboBox" name="rebatePage_sortBox">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>10</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QToolButton" name="rebatePage_refreshButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ðŸ—˜</string>
     </property>
    </widget>
    <widget class="QTableView" name="rebatePage_tableView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>761</width>
       <height>541</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="inventoryButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans</family>
    </font>
   </property>
   <property name="text">
    <string>Inventory List</string>
   </property>
  </widget>
  <widget class="QPushButton" name="salesButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sales Info</string>
   </property>
  </widget>
  <widget class="QPushButton" name="membershipButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans</family>
    </font>
   </property>
   <property name="text">
    <string>Memberships</string>
   </property>
  </widget>
  <widget class="QPushButton" name="customerButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Customer Details</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rebatesButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans</family>
    </font>
   </property>
   <property name="text">
    <string>View Rebates</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>inventoryPage_searchBar</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Manager</receiver>
   <slot>refreshSearch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refreshSearch()</slot>
  <slot>generateReport()</slot>
 </slots>
</ui>
